app/views/about/show.html.haml

```diff
@@ -19,9 +19,12 @@
     %h3 Innovate
     %p
       Use this opportunity to work on any activity of your passion. You decide
-      whether you contribute to an existing free software project or explore
-      strange new languages. You choose whether you seek out new tools and new
-      communities, or boldly go where no hacker has gone before! Just
+      whether you contribute to
+      = link_to projects_path(@episode) do
+        an existing Hack Week project
```

Comment:
I don't like the change of meaning. This paragraph is about choosing what you want to do during hack week, not choosing how to do it (create a HW project / join a HW project).